csrc/scheduler/resize.cpp

```diff
@@ -251,6 +263,17 @@ void ResizeScheduler::schedule(Fusion* fusion, const HeuristicParams* params) {
     ir_utils::replaceValInExprInputs(resize_tensor_op, inp_tv, inp_tv_copy);
   }
 
+  TensorView* largest_input = nullptr;
+  if (resize_params->largest_input >= 0) {
+    largest_input =
+        fusion->inputs().at(resize_params->largest_input)->as<TensorView>();
+
+    // The tensors are going to be reordered to align with the largest
+    // input. To make it work, merge operations for reshape should be
+    // cancelled.
+    scheduler_tools::cancelReshapeInLoopDomains(largest_input);
```

Comment:
I wonder what would happen if we have transformation that can't be cancelled. https://github.com/NVIDIA/Fuser/blob/3d27e10409568d62df3435a89ff4a371c12d7b81/csrc/scheduler/tools/loop_domain_scheduler.h#L102-L104

Is this reshape cancel applied here, so that later when we call `scheduler_tools::scheduleLoopDomainsLike` from the reference would successfully apply?
To re-phrase the question, are we expecting cancel Reshape to successfully cancel transformations on *all* tensors, or we are just trying to cancel the transformation on *reference tv*?

Reply:
All tensors that are direct or indirect consumers of `largest_input` will be targets of the cancellation. Reshapes are not just done on `largest_input`. Any reshape op that depends on `largest_input`, directly or indirectly, will be cancelled.

If it isn't valid, it'll be just skipped, so it's done in a best-effort manner. It should not affect the correctness but the reference reordering may become suboptimal.

To be honest, this is only tested with the RoPE patterns, so it's likely there would be some corner cases, which I'd probably need to work on before enabling the resize scheduler by default.